src/game/utils/score.c

```diff
@@ -38,6 +39,8 @@ vec2i interpolate(vec2i start, vec2i end, float fraction) {
 }
 
 void chr_score_create(chr_score *score) {
+    score->text_cache[0].dynamic = true;
```

Comment:
Unclear why this needs to be dynamic. Only the x/y position changes, maybe those should not be cached?

Reply:
Dynamic means something special, it means that the frame-over-frame rendering of the text object cannot be guaranteed to be the same. As the object may get occluded and require a re-render. Unfortunately I couldn't figure out a way to track this, so I resorted to hand picking objects that were NOT-SAFE to copy from the previous frame.

Reply:
This only really matters on the N64, where we can copy from the previous frame buffer, instead of allocating an intermediate texture.
The fallback would be to allocate the intermediate texture on the N64.
And can be completely ignored on the PC.

Reply:
ah, ok, maybe we can have a better name than dynamic, or have a comment that explains it.

Reply:
other names I've considered: not_copy_safe, needs_intermediate_texture, not_static_composition,

Reply:
is `uncachable` or `changes_frequently` appropriate?

Reply:
It's more of a hint to the renderer telling it not to 2nd level optimize.
0th lvl - render each letter.
1st lvl - allocate a new texture, render each letter to texture on dirty bit, use the allocated texture on all invocations without the dirty bit.
2nd lvl - render each letter to FB on dirty bit, copy from FB without dirty bit.